addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -88,3 +89,39 @@ def test_notify_reply_to_computation_mc(self):
                     "%s %s" % (self.user_employee_c2.company_id.name, test_record.name),
                     "%s@%s" % (self.alias_catchall, self.alias_domain)))
             )
+
+@tagged('multi_company')
+class TestMultiCompanySetupAttachment(TestMailCommon, TestRecipients):
+    """ Test message post with attachment on record of another company than the
+    user company but for which the user has readonly access on. """
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._activate_multi_company()
+
+        cls.test_record = cls.env['mail.test.simple.mc'].with_context(cls._test_context).create({
```

Comment:
A model 'mail.test.multi.company' already exists, can't we re-use for this test ? Seems it fulfills the same "simple model" target, as it has the same rule company-based than the new one you are introducing here. Did I miss something ?